### ISSUE TITLE
import/order triggered only after adding newline

### ISSUE THREAD

**Issue by koutsenko** [benmosher/eslint-plugin-import#1711]
Steps to reproduce
1. Install create-react-app
2. Add import/order rule with groups via EXTEND_ESLINT
3. Run it - no errors occured
4. Add newline before ```function App() {``` (2 newline between import and function now)
5. Save - now error occured 
```
./src/App.js
  Line 4:1:  `./logo.svg` import should occur before import of `react`  import/order

Search for the keywords to learn more about each error.
```

Or 
1. Checkout https://github.com/koutsenko/sort-imports
2. Run it - it will print error
3. Remove one of two newlines between import and function statements
4. Import error will gone.

*Question*: why import rules are triggered by adding newline?

**Comment by koutsenko** [comment]
@ljharb looks like create-react-app eslint cache also preventing to debug this issue well. At least, can you reproduce and confirm the issue?

**Comment by ljharb** [comment]
I'm occupied with TC39 this week; the fastest approach would be to make a PR with a test case that fails for you locally, and we can see what CI does.

**Comment by koutsenko** [comment]
@ljharb  Can i see an example of test? Never writes them but i can try

**Comment by ljharb** [comment]
The "order" rule's tests are here: https://github.com/benmosher/eslint-plugin-import/blob/master/tests/src/rules/order.js#L13
